addons/account/models/account_bank_statement.py

```diff
@@ -442,19 +442,23 @@ def _prepare_reconciliation_move_line(self, move, amount):
         st_line_currency = self.currency_id or statement_currency
         amount_currency = False
         st_line_currency_rate = self.currency_id and (self.amount_currency / self.amount) or False
+        if isinstance(move, dict):
```

Comment:
this part needs to be documented
* in the docstring (still referencing `move` as a recordset)
* here and at line 482